particles/Tutorial/Javascript/Demo/source/Game.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+'use strict';
+
+/* global defineParticle */
+// TODO (heimlich@) change to particle
```

Comment:
? I don't understand this comment

Reply:
It was to change it to simple particle :)